pkg/actions/cluster/delete_test.go

```diff
@@ -0,0 +1,127 @@
+package cluster
+
+import (
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/mock"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager/fakes"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/kubernetes"
+	"github.com/weaveworks/eksctl/pkg/testutils/mockprovider"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+type drainerMock struct {
+	mock.Mock
+}
+
+func (drainer *drainerMock) Drain(nodeGroups []eks.KubeNodeGroup, plan bool, maxGracePeriod, nodeDrainWaitPeriod time.Duration, undo, disableEviction bool) error {
+	args := drainer.Called(nodeGroups, plan, maxGracePeriod, nodeDrainWaitPeriod, undo, disableEviction)
+	return args.Error(0)
+}
+
+var _ = Describe("Delete", func() {
```

Comment:
```suggestion
var _ = Describe("DrainAllNodeGroups", func() {
```

Reply:
Changed for better context as well, thanks!